RTN-004.tex

```diff
@@ -53,7 +57,320 @@
+
+\subsection{Responsibilities}\label{ssec:del_resp}
+
+The most important responsibility is that DP0 delegates learn how to use the RSP via the on-boarding process well enough to perform some basic scientific exercises, and then take action that informs or enhances the DMS, the RSP, or a Science Collaboration. 
+
+Such actions are envisioned to be simple and optional (they will not be tracked), and it will be left to the discretion of individual delegates to decide what kind of actions best match their interests and expertise.
+Examples of delegate actions are provided in Section \ref{ssec:res_act}.
+
+The time spent by DP0 delegates will not be tracked and there is no minimum (or maximum), but the estimated time commitment for DP0 delegates to gain at least some of benefits of participation is approximately 12-36 hours (e.g., 1-3 hours per week over three months).
+This time can be distributed in any manner that is convenient to the delegate, and allocated accounts do not expire\footnote{In the case of oversubscription for the 300 RSP accounts, delegates might be asked to relinquish their accounts if they are not using them, in order to make way for others (Section \ref{ssec:res_pol}).}.
+
+In addition to the above, all DP0 delegates will be expected to:
+\begin{itemize}
+\item abide by the data rights policy in \citeds{RDO-013}
+\item adhere to security policies regarding their RSP accounts at the IDF
+\item abide by usage policies of the RSP and the IDF
+\item use the on-boarding resources to learn about the RSP
+\item follow the off-boarding steps if/when appropriate
+\item engage in DP0-related discussions via the Community Forum when possible
+\item help other DP0 delegates resolve issues when possible
+\end{itemize}
+
+Policies regarding account usage, security, and off-boarding are described in Section \ref{ssec:res_pol}, and on-boarding resources for DP0 delegates in Section \ref{ssec:res_sup}. 
+
+
+\section{Delegate Selection Process}\label{sec:sel}
+
+There will be 300 RSP user accounts at the IDF available for science community DP0 delegates, in addition to the ~150 accounts for Rubin staff members.
+This marks an almost order of magnitude increase in the number of community accounts in the current NCSA instance of the RSP (via the Stack Club).
+
+The number of RSP accounts remains constrained during the Rubin Observatory construction and pre-operations phase -- and especially for DP0 -- due to the Rubin pre-operations team's limited scope in providing support for data software and services that are still in development.
+Time is needed to scale up in a sustainable manner.
+At least by the start of Rubin Observatory Operations, all data rights holders will be able to have RSP accounts.
+It is a primary goal of the Rubin pre-operations team to provide RSP access as soon as possible to all data rights holders. 
```

Comment:
"as soon as possible" is a dangerous phrase and needs amendment/clarification. For example, it can be interpreted to mean as soon as it is technically possible, but that is not the case. For example, if it was technically possible to open the science platform to all data rights holders (which remember, we don't even have a real estimate for) in the summer, the pre-ops science platform team is only 25% staffed and therefore is in no position to support a fully scaled operational service (if it did, we would have lied about how much effort we need in ops). I suggest a much more realistic phrasing, such as "it is a primary goal ... to provide RSP access to all data rights holders in time for the community to be well prepared for {Data Release Blah}

Reply:
This phrasing came from OpsExec (or at least, I think it did?) and I'm not exactly sure what might be more appropriate to say here, so am bringing this comment to the attention of @leannep and @rblum5.